aliasmgr/aliasmgr.go

```diff
@@ -302,8 +303,9 @@ func (m *Manager) AddLocalAlias(alias, baseScid lnwire.ShortChannelID,
 	// Update the aliasToBase and baseToSet maps.
 	m.baseToSet[baseScid] = append(m.baseToSet[baseScid], alias)
 
-	// Only store the gossiper map if gossip is true.
-	if gossip {
+	// Only store the gossiper map if gossip is true, or if the caller
+	// explicitly asked to store this reverse mapping.
+	if gossip || baseLookup {
```

Comment:
i dont think we can do this cause now it will be stored in the `m.aliasToBase` map if `gossip` is false but `baseLookup` is true.. meaning it will be available in the map when the gossiper calls `FindBaseByAlias` which we explicitly dont want, right?